src/main/resources/languages/english.yml

```diff
@@ -35,6 +35,18 @@ Items:
     Name: "Energy Drink"
   Handcuffs:
     Name: "Handcuffs"
+  PlasticSpoon:
+    Name: "Wooden Shovel"
+  MetalSpoon:
+    Name: "Golden Shovel"
```

Comment:
Here as well

Reply:
It is supposed. Since we are using weaker shovels to represent spoons I think it will be good to name it shovel as well. This way players won't be confused. Either way that is just the config name so it can easily be changed later.